client/src/components/JobDescription.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { Grid, Typography, Link } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import KeyboardArrowUpOutlinedIcon from "@mui/icons-material/KeyboardArrowUpOutlined";
+import { Link as RouterLink } from "react-router-dom";
+
+const JobDescription = (props) => {
+  const [ReadMore, setReadMore] = useState(false);
+  const toggle = () => {
+    setReadMore(!ReadMore);
+  };
+  const centered = {
+    display: "flex",
+    justifyContent: "center",
+    height: "3vh",
+  };
+  const FullDescription = ({ children }) => (
```

Comment:
https://legacy.reactjs.org/docs/composition-vs-inheritance.html

You will want to get familiar with the concept of composition when working with react function components